### PR TITLE
check in the 2nd tutorial

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Help users understand what model deployment is, what MMDeploy does and how MMDeploy does it

## Modification
docs/zh_cn/tutorials/chapter_02_problems.md
docs/zh_cn/index.rst

### CODE REVIEW

File: docs/zh_cn/tutorials/chapter_02_problems.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+        out = self.relu(self.conv1(x))
+        out = self.relu(self.conv2(out))
+        out = self.conv3(out)
+        return out
+
+
+def init_torch_model():
+    torch_model = SuperResolutionNet()
+
+    state_dict = torch.load('srcnn.pth')['state_dict']
```
Comment (AllentDan) [inline docs/zh_cn/tutorials/chapter_02_problems.md:75]: Consider the user may read this tutorial ahead of tutorial 1. `srcnn` and `face.png` maybe a little confusing

**Review by AllentDan — APPROVED** [review]
LGTM

File: docs/zh_cn/tutorials/chapter_02_problems.md (hunk 1 above)
Comment (lvhan028) [reply docs/zh_cn/tutorials/chapter_02_problems.md:77]: > Consider the user may read this tutorial ahead of tutorial 1. `srcnn` and `face.png` maybe a little confusing

I added comments above this line

File: docs/zh_cn/index.rst [hunk 2]
```diff
@@ -15,6 +15,7 @@
    :caption: 教程
 
    tutorials/chapter_01_introduction_to_model_deployment.md
+   tutorials/chapter_02_problems.md
```
Comment (SingleZombie) [inline docs/zh_cn/index.rst:18]: How about renaming "problems" to "chanllenges"?

**Review by SingleZombie — APPROVED** [review]
LGTM

File: docs/zh_cn/index.rst (hunk 2 above)
Comment (lvhan028) [reply docs/zh_cn/index.rst:18]: > How about renaming "problems" to "chanllenges"?

Sure, done

**Pull request merged**